pkg/scheduler/grb_email_jobs.go

```diff
@@ -0,0 +1,67 @@
+package scheduler
+
+import (
+	"context"
+
+	"github.com/go-co-op/gocron/v2"
+	"go.uber.org/zap"
+
+	"github.com/cms-enterprise/easi-app/pkg/appcontext"
+	"github.com/cms-enterprise/easi-app/pkg/email"
+	"github.com/cms-enterprise/easi-app/pkg/logfields"
+	"github.com/cms-enterprise/easi-app/pkg/storage"
+)
+
+type grbEmailJobs struct {
+	// SendAsyncVotingHalfwayThroughEmailJob is a job that sends an email when the voting session is halfway through
+	SendAsyncVotingHalfwayThroughEmailJob ScheduledJob
+}
+
+var GRBEmailJobs = GetGRBEmailJobs(GetScheduler())
+
+// &grbEmailJobs{
+// 	SendAsyncVotingHalfwayThroughEmailJob: NewScheduledJobWrapper(sharedScheduler, gocron.CronJob("0 2 * * *", false), sendAsyncVotingHalfwayThroughEmailJobFunction),
+// }
+
+// GetGRBEmailJobs initializes all GRB email jobs
+func GetGRBEmailJobs(scheduler gocron.Scheduler) *grbEmailJobs {
+	return &grbEmailJobs{
+		SendAsyncVotingHalfwayThroughEmailJob: NewScheduledJob("SendAsyncVotingHalfwayThroughEmailJob", scheduler,
+			//  gocron.CronJob("0 2 * * *", false),
+			//this is for testing so that it runs every 5 seconds
+			gocron.CronJob("*/5 * * * * *", true),
```

Comment:
commented code still relevant?

Reply:
thanks Tom, yes this is just like this for testing. Though from your other feedback, there probably is some flexibility to define some timings in there own file and reuse if applicable.

Reply:
refactored!